components/error_code/bin.rs

```diff
@@ -0,0 +1,30 @@
+use std::{fs, io::Write, path::Path};
+
+use error_code::*;
+
+fn main() {
+    let err_codes = vec![
+        codec::ALL_ERROR_CODES.iter(),
```

Comment:
Are these `.iter()` necessary?

Reply:
`lazy_static` generates each static variable as a new type, need to convert them to a uniform type.